components/eamxx/cime_config/buildnml

```diff
@@ -20,13 +20,13 @@ Some import errors are expected and can be ignored
 
 It is encouraged that any changes to this system be tested with
 eamxx/cime-nml-tests
-
 """
 
-import os
+import os, sys
```

Comment:
Not really important, but is sys needed at all?

Reply:
Interesting. I only looked at changes, and didn't see anything with sys. How did this manage to work before this fix then?